### PR TITLE
Add comment

### PR DESCRIPTION
**Pull request by mattvv**
Simple adding of Comments to stories

### CODE REVIEW

File: components/AddComment.tsx [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+    await PivotalHandler.addComment({ apiKey, projectId: id, storyId: story.id, text: comment });
+    const newStory = await PivotalHandler.fetchStory({
+      apiKey,
+      projectId: id,
+      storyId: story.id,
+    });
+    dispatch(editStory({ projectId: id, story: newStory, storyState: newStory.current_state }));
+    setComment('');
+    setLoading(false);
+  };
```
Comment (jansen0825) [inline components/AddComment.tsx:35]: Any reason why not using useAsync?

**Review by jansen0825 — APPROVED** [review]
Looking good!

**Review by dervin10 — APPROVED** [review]
Nice Implementation 💯

File: components/AddComment.tsx (hunk 1 above)
Comment (mattvv) [reply components/AddComment.tsx:35]: Didnt know we had it in this project! Feel free to update this for me :)

**Review by shc023 — APPROVED** [review]
👍

**Pull request merged**